### PR TITLE
Support for multiple SPS, resolution change at SPS

### PR DESCRIPTION
**Pull request by p-chirag**
@mpeg5 - Added fix for supporting multiple sps, resolution change at sps.

### CODE REVIEW

File: src_base/xevd.c [hunk 1]
```diff
@@ -1645,7 +1652,7 @@ void xevd_flush(XEVD_CTX * ctx)
 int xevd_dec_nalu(XEVD_CTX * ctx, XEVD_BITB * bitb, XEVD_STAT * stat)
 {
     XEVD_BSR  *bs = &ctx->bs;
-    XEVD_SPS  *sps = &ctx->sps;
+    XEVD_SPS  *sps = &ctx->sps_array[ctx->sps_id];
```
Comment (kpchoi) [inline src_base/xevd.c:1655]: is it safe to use ctx->sps_id without checking whether the ctx->sps_id is existed or not?

Comment (kpchoi) [reply src_base/xevd.c:1655]: Ah, ctx->sps_id is not actual syntax value. 
it seems to be current maximum sps id?

Comment (kpchoi) [reply src_base/xevd.c:1655]: the variable name seems to be confuzed.

Comment (p-chirag) [reply src_base/xevd.c:1655]: I think we need a assert after incrementing sps_id, not to go beyond 16

**Review by kpchoi — APPROVED** [review]
LGTM

**Pull request merged**